options/flags.go

```diff
@@ -168,10 +170,18 @@ func (o *Options) AddFlags(cmd *cobra.Command) {
 		fmt.Sprintf("Checks to run. Possible values are: %s", strings.Join(checkNames, ",")),
 	)
 
+	cmd.Flags().StringSliceVar(
+		&o.ProbesToRun,
+		FlagProbes,
+		o.ProbesToRun,
+		"Probes to run.",
+	)
+
 	// TODO(options): Extract logic
 	allowedFormats := []string{
 		FormatDefault,
 		FormatJSON,
+		FormatPJSON,
```

Comment:
can we remove PJSON from the allowed formats, so it remains gated behind the experimental flag for now?

Reply:
Removed in https://github.com/ossf/scorecard/pull/3688/commits/829fc436996e8579296a8e3a13e142b3eda76f3b